config/locales/forms/register_in_northern_ireland_forms/en.yml

```diff
@@ -1,6 +1,7 @@
 en:
   register_in_northern_ireland_forms:
     new:
+      title: You should register in Northern Ireland
```

Comment:
I'd edge towards dropping the **You should** in the titles of these pages, so go with `Register in Northern Ireland` instead.

What do you think?